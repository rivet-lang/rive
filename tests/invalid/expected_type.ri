fn my_fn() void { }

fn x(a: my_fn) void {}

fn main() void { }
