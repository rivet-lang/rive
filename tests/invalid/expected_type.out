tests/invalid/expected_type.ri:3:9: error: expected type, found function
  3 | fn x(a: my_fn) {}
    |         ^
rivetc: error: could not compile module `expected_type`, aborting due to previous error